operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -499,28 +488,32 @@ def validateNonDynamicClusterServersHaveSameListenPort(self, cluster):
+                           + ".  All ports for the same channel in a cluster must be the same and they must either be "
+                           + "enabled or disabled uniformly")
           if sslListenPortEnabled != firstSslListenPortEnabled:
-            self.addError("The WebLogic configured cluster " + self.name(cluster) + "'s server " + self.name(firstServer) + " has ssl listen port enabled: " + self.booleanToString(firstSslListenPortEnabled) + " but its server " + self.name(server) + "'s ssl listen port enabled: " + self.booleanToString(sslListenPortEnabled) + ".  Channels in a cluster must be either all enabled or disabled.")
+            self.addError("The WebLogic configured cluster " + self.name(cluster) + "'s server " + self.name(firstServer) + " has ssl listen port enabled: " + self.booleanToString(firstSslListenPortEnabled) + " but its server " + self.name(server) + "'s ssl listen port enabled: " + self.booleanToString(sslListenPortEnabled)
+                          + ".  All ports for the same channel in a cluster must be the same and they must either be "
+                          + "enabled or disabled uniformly")
           if adminPort != firstAdminPort:
             self.addError("The WebLogic configured cluster " + self.name(cluster) + "'s server " + self.name(firstServer) + "'s ssl listen port is " + str(firstAdminPort) + " but its server " + self.name(server) + "'s ssl listen port is " + str(adminPort) + ".  All ports for the same channel in a cluster must be the same.")
           if adminPortEnabled != firstAdminPortEnabled:
             self.addError("The WebLogic configured cluster " + self.name(cluster) + "'s server " + self.name(firstServer) + " has ssl listen port enabled: " + self.booleanToString(firstAdminPortEnabled) + " but its server " + self.name(server) + "'s ssl listen port enabled: " + self.booleanToString(adminPortEnabled) + ".  Channels in a cluster must be either all enabled or disabled.")
 
-  def validateClusterServersListenPortProperty(self, cluster, errorMsg, clusterListenPortProperty):
-    firstServer = None
-    firstListenPortProperty = None
-    for server in self.env.getDomain().getServers():
-      if cluster is self.env.getClusterOrNone(server):
-        listenPortProperty = getServerClusterPortPropertyValue(self, server, clusterListenPortProperty)
-        if firstServer is None:
-          firstServer = server
-          firstListenPortProperty = listenPortProperty
-        else:
-          if listenPortProperty != firstListenPortProperty:
-            self.addError(errorMsg.substitute(cluster=self.name(cluster), server1=self.name(firstServer), property=clusterListenPortProperty, value1=str(firstListenPortProperty), server2=self.name(server), value2=str(firstListenPortProperty)))
-            return
+  # def validateClusterServersListenPortProperty(self, cluster, errorMsg, clusterListenPortProperty):
```

Comment:
Let's remove commented out code unless there is a specific need, e.g. it acts as documentation. In this case, it looks like it's just leftover after the other changes.

Reply:
Done